src/search/state_registry.h

```diff
@@ -177,28 +181,28 @@ class StateRegistry : public subscriber::SubscriberService<StateRegistry> {
         return num_variables;
     }
 
-    int get_state_value(const PackedStateBin *buffer, int var) const {
```

Comment:
We could leave this like it is but that would mean that the `State` class cannot store a pointer to the state packer and has to be aware of the implementation of `StateRegistry` to access packed states. Because of a cyclic dependency in the headers, this would mean that we cannot inline `State::operator[]` and `State::unpack".